<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>548</width>
     <height>330</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>50</number>
    </property>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink6">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/6.png</normaloff>
          <normalon>drink_images/6_c.png</normalon>drink_images/6.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/2.png</normaloff>
          <normalon>drink_images/2_c.png</normalon>drink_images/2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink4">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/4.png</normaloff>
          <normalon>drink_images/4_c.png</normalon>drink_images/4.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink0">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/0.png</normaloff>
          <normalon>drink_images/0_c.png</normalon>drink_images/0.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink1">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/1.png</normaloff>
          <normalon>drink_images/1_c.png</normalon>drink_images/1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink3">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/3.png</normaloff>
          <normalon>drink_images/3_c.png</normalon>drink_images/3.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink5">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/5.png</normaloff>
          <normalon>drink_images/5_c.png</normalon>drink_images/5.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_drink7">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>drink_images/7.png</normaloff>
          <normalon>drink_images/7_c.png</normalon>drink_images/7.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>85</width>
          <height>85</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="styleSheet">
         <string notr="true">font-weight:600</string>
        </property>
        <property name="text">
         <string>제품</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>가격</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>제품</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>시스템 메시지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>379</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>수량</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>379</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>금액</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>320</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_purchase">
      <property name="text">
       <string>구입</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_insert">
      <property name="text">
       <string>입금</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_return">
      <property name="text">
       <string>잔돈 반환</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_director">
      <property name="text">
       <string>관리자 모드</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stock">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cost">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFFFFF;
border-width: 1px;
border-style:outset;
min-width:1%;
min-height:2%;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
